app/services/doc_auth/lexis_nexis/requests/true_id_request.rb

```diff
@@ -23,6 +23,13 @@ def initialize(
           @liveness_checking_required = liveness_checking_required
         end
 
+        def request_context
+          {
+            settings: settings,
```

Comment:
Are these settings being used? What for?

Reply:
@charleyf , not now, there are reference id etc that can be used for logging etc.

Reply:
@charleyf , using it to get reference of the request, it was fetched from response where it is not always available(like 500 etc).